src/streamlink/plugins/cmmedia.py

```diff
@@ -0,0 +1,121 @@
+"""
+$description Live TV channel and video on-demand service from CMM, a Spanish public, state-owned broadcaster.
+$url cmmedia.es
+$type live, vod
+$notes Content not licensed for digital distribution is unavailable via the live channel stream.
```

Comment:
Are you saying you believe it's geo-blocked because you're seeing the red-ish screen with the text you mentioned above?  I was just about to screen-shot it, but right now it's broadcasting for me from a UK IP address.  If you're still awake right now, I wonder if you can see it too?

I wrote this before I went to screen-shot the message I think you were talking about:

I'm not sure about this yet.  Let me try checking with a proxy to see if there's any difference.  I have seen programming appear on the live stream from a UK IP address, so I have taken the message to mean what I have described in `$notes` here and that it would be the same even from a Spanish IP address because the operators have not negotiated rights for online distribution of the content.

Reply:
No placeholder video...

Reply:
Weird, I've spun up a VM for use as a proxy in Frankfurt and it's streaming content as I type this, although it was showing the placeholder video earlier with my UK IP address.

So, I'm not sure if it's the case I have described above or actual geo-blocking.

Reply:
So, I spun up a VM in Madrid for use as a proxy and that can also display the placeholder video, so I don't think it's geo-blocking, just a content rights/usage thing.

Reply:
I'll change this text if you have any thoughts on it?

Reply:
No, should be fine the way it is...